Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -9,10 +9,11 @@ import SwiftUI
 
 struct DashboardView: View {
     
-    @StateObject private var viewModel: DashboardViewModel
+    @State private var isShowingAddView = false
+    @Bindable private var viewModel: DashboardViewModel
```

Comment:
I think the `DashboardViewModel` needs to be `@State`

Reply:
No, here we need `@Bindable` (see the link @elmoritz provided)